.vscode/launch.json

```diff
@@ -8,7 +8,11 @@
       "name": "Run single test",
       "program": "${workspaceFolder}/${relativeFile}",
       "cwd": "${workspaceFolder}",
-      "runtimeExecutable": "${workspaceFolder}/node"
+      "runtimeExecutable": "${workspaceFolder}/node",
+      "skipFiles": [
+        // VSCode excludes node internals by default. Uncomment this to include them.
```

Comment:
I don't think vscode skips `<node_internals>` by default. The document also suggests that the config has to be manually added: https://code.visualstudio.com/docs/nodejs/nodejs-debugging#_additional-configuration